product/swagger/product.yaml

```diff
@@ -296,4 +296,63 @@ paths:
           description: Produto não encontrado
           content:
             application/json:
-              example: Este produto não foi encontrado
\ No newline at end of file
+              example: Este produto não foi encontrado
+  /api/products/listByPrice:
+    parameters:
+    - name: max
+      in: query
+      schema:
+        type: number
+      required: true
+    - name: min
+      in: query
+      schema:
+        type: number
+      required: true   
+    get:
+      summary: Lista os produtos por faixa de preço
+      responses:
+        200:
+          description: Sucesso
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/products"
+        404:
+          description: Produtos não encontrados
+  /api/products/listByCategory:
+    parameters:
+    - name: category
+      in: query
+      schema:
+        type: string
+      required: true
+    get:
+      summary: Lista os produtos da categoria determinada
+      responses:
+        200:
+          description: Sucesso
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/products"
+        404:
+          description: Produtos não encontrados
+  /api/products/listByStock:
+    parameters:
+    - name: stock
+      in: query
+      schema:
+        type: number
+      required: true
+    get:
+      summary: Lista os produtos conforme seu estoque
+      responses:
+        200
```

Comment:
missing indentation

and take a look at the references

Reply:
thanks for reviewing, I forgot my schemas were written in portuguese. I also fixed indentation and added a tag Produto on the last three verbs